docs/it/metadata_oidc_rp.rst

```diff
@@ -60,6 +60,12 @@ Il Metadata di tipo **"openid_relying_party"** DEVE contenere almeno i seguenti
   * - **id_token_signed_response_alg**
     - Vedi `OpenID.Registration#ClientMetadata`_.
     - |spid-icon| |cieid-icon| 
+  * - **id_token_encrypted_response_alg**
+    - Vedi `OpenID.Registration#ClientMetadata`_. Opzionale. Se questo parametro è incluso l'ID Token restituito sarà un *nested JWT* firmato e cifrato come definito in :rfc:`7515#section-4`.
+    - |spid-icon| |cieid-icon| 
+  * - **id_token_encrypted_response_enc**
+    - Vedi `OpenID.Registration#ClientMetadata`_. Obbligatorio solo nel caso sia presente anche il parametro *id_token_encrypted_response_alg*.
```

Comment:
Obbligatorio se il parametro *id_token_encrypted_response_alg* è presente.

Reply:
fixed in https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/8ec33eca96a7075ce9c222eb16e07b2fc6820176